client/end.py

```diff
@@ -0,0 +1,29 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen):
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    back_to_login = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    REJOIN = False
+    while not done:
+        for event in pg.event.get():
+            if event.type == pg.QUIT:
+                return SHOULD_STOP_GAME
```

Comment:
for the end screen, returning this is incorrect. Recall that the end screen returns either rejoin, don't rejoin, or None. The None should go here